### PR TITLE
Adding test to check Tag and list images endpoints in apis.

### PR DESCRIPTION
**Pull request by sujil02**
Include test cases to validate tag api responses and check if 
all the image instances are shown list command.
Also, Update the binding process and the response

Signed-off-by: Sujil02 <sushah@redhat.com>

### CODE REVIEW

**Comment by baude** [comment]
@sujil02 I will help you in the morning to resolve the gating test failure.  Welcome to your first PR to Podman!

**Comment by rhatdan** [comment]
@baude is removing the exit codes ok?  I thought that was used as input for generating the API Docs?

**Comment by baude** [comment]
those were only for dev as reminders.

File: pkg/bindings/test/images_test.go [hunk 1]
```diff
@@ -89,4 +90,45 @@ var _ = Describe("Podman images", func() {
 
 	})
 
+	//Tests to validate the image tag command.
+
```
Comment (mheon) [inline pkg/bindings/test/images_test.go]: Extra whitespace here

File: pkg/bindings/test/images_test.go [hunk 2]
```diff
@@ -89,4 +90,45 @@ var _ = Describe("Podman images", func() {
...
+		Expect(err).To(BeNil())
+
+		//Validates if name updates when the image is retagged.
+		_, err := images.GetImage(connText, "alpine:demo", nil)
+		Expect(err).To(BeNil())
+
+	})
+
+	//Test to validate the List images command.
+	FIt("List image", func() {
```
Comment (baude) [inline pkg/bindings/test/images_test.go]: need tro remove F

File: pkg/bindings/test/images_test.go [hunk 3]
```diff
@@ -89,4 +90,44 @@ var _ = Describe("Podman images", func() {
...
+		_, err := images.GetImage(connText, "alpine:demo", nil)
+		Expect(err).To(BeNil())
+
+	})
+
+	//Test to validate the List images command.
+	It("List image", func() {
+
+		//Array to hold the list of images returned
+		var imageSummary []*handlers.ImageSummary
```
Comment (baude) [inline pkg/bindings/test/images_test.go]: rm this line

File: pkg/bindings/test/images_test.go [hunk 4]
```diff
@@ -89,4 +90,44 @@ var _ = Describe("Podman images", func() {
...
+		imageSummary, err = images.List(connText, nil, nil)
```
Comment (baude) [inline pkg/bindings/test/images_test.go]: use := instead of =

File: pkg/bindings/test/images_test.go [hunk 5]
```diff
@@ -89,4 +90,44 @@ var _ = Describe("Podman images", func() {
...
+
```
Comment (baude) [inline pkg/bindings/test/images_test.go]: check that err is nil

**Comment by TomSweeneyRedHat** [comment]
LGTM and happy green test buttons

**Comment by rhatdan** [comment]
/lgtm
@sujil02 Nice job.

File: pkg/bindings/test/images_test.go [hunk 6]
```diff
@@ -89,4 +89,39 @@ var _ = Describe("Podman images", func() {
 
 	})
 
+	//Tests to validate the image tag command.
```
Comment (haircommander) [inline pkg/bindings/test/images_test.go]: hey @sujil02 this is a super pedantic nit, and can be addressed later, but typically I prefer to have a space between the `//` and the comment, and the rest of libpod conforms to this. Could you make an update PR to fix these instances?

Comment (sujil02) [reply pkg/bindings/test/images_test.go]: Hi,

Thank you for letting me know, I am working on a PR to add more test cases in the same file. And I will take care of this along with that PR.

**Pull request merged**